package-lock.json

```diff
@@ -649,6 +657,19 @@
         "@jridgewell/sourcemap-codec": "^1.4.10"
       }
     },
+    "node_modules/@digitalbazaar/http-client": {
```

Comment:
@vmidyllic Is dependency ok?

Reply:
@vmidyllic  I think we use this dependency in mineralization flow
https://github.com/iden3/polygonid-js-sdk/blob/main/package-lock.json#L691-L703

Reply:
As I understand it's ok for commercial usage.  @OBrezhniev  @demonsh check it.
https://choosealicense.com/licenses/bsd-3-clause/

Reply:
Yes, BSD-3-Clause license is ok.